backend/director/tools/videodb_tool.py

```
@@ -396,3 +478,90 @@ def delete_image(self, image_id):
+        os.makedirs(DOWNLOADS_PATH, exist_ok=True)
+
+        response = requests.get(video_url, stream=True)
+        response.raise_for_status()
+
+        if not response.headers.get('Content-Type', '').startswith('video'):
+            raise ValueError(f"The URL does not point to a video file: {video_url}")
+
+        with open(save_at, 'wb') as file:
+            file.write(response.content)
+
+    def text_to_video(self, prompt: str, save_at: str, duration: float, config: dict = {}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)